lib/fluent/plugin/in_tail/position_file.rb

```diff
@@ -224,5 +254,7 @@ def read_inode
         @inode
       end
     end
+
+    PathInodeTuple = Struct.new(:path, :ino)
```

Comment:
I'm re-consindering this struct name. This name depends on members.
If we add more member in the future, this name will be invalid.
So `TargetInfo` or other name is better for the future.

Reply:
Fixed in https://github.com/fluent/fluentd/pull/3182/commits/72b9d4c6c7e0505e51da2f1f5024056bf71ef6c7.